Drive/Features/TrackDrive/DriveVC.swift

```diff
@@ -55,12 +55,29 @@ class DriveVC: UIViewController, StoryboardInstantiable {
         super.viewDidLoad()
         setUpCollectionView()
         setUpDataSource()
-        setUpSnapshot()
+        loadDrives()
```

Comment:
Can `loadDrives()` be called after `setUpLayout()`? Highly unlikely but mainly thinking about the request coming back before the layout of the table has been set. Might run into this issue when testing but 🤷🏽‍♂️